src/Components/SmartComponents/InsightsSystemRule/InsightsSystemRule.test.js

```diff
@@ -66,7 +66,7 @@ describe('InsightsSystemRule', () => {
 
     it('Should render empty message', () => {
         const wrapper = mountWithIntl(<InsightsSystemRule rule={null} cve="CVE_2018_12130"/>);
-       expect(wrapper.text()).toBe("This system is affected by CVE_2018_12130, but is not affected by any security rules associated with this CVE.Security rules are written by Red Hat to help you configure your systems, and are denoted with an icon: Learn more about security rules")
+       expect(wrapper.text()).toBe("This system is affected by CVE_2018_12130, but is not affected by any security rules associated with this CVE.Security rules are written by Red Hat to help you configure your systems, and are denoted with:Security ruleLearn more about security rules")
```

Comment:
`Security ruleLearn` no space here.

Reply:
"Security rule" is coming from inside the label, thats why there is no space.
![Screenshot from 2021-02-03 15-49-07](https://user-images.githubusercontent.com/8426204/106763739-6fcef280-6637-11eb-9864-6a7f15346864.png)